tests/proptest-regressions/native_vp/pos.txt

```diff
@@ -1,2 +1,2 @@
 cc 65720acc67508ccd2fefc1ca42477075ae53a7d1e3c8f31324cfb8f06587457e
```

Comment:
looks like something got corrupted here - perhaps there was a missing new line. I'll drop the new seed

Reply:
done in dda4008efd69eac43b0daa62f0094042e5fae152